karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/oasExamples.feature

```diff
@@ -0,0 +1,410 @@
+@sandbox-only
+Feature: Update patient's record - OAS file examples
+    The public OAS file (https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir) lists a number
+    of search examples and expected responses. This feature file makes sure the sandbox behaves in the same way that the
+    documentation describes.
+
+  Background:
+    * def utils = call read('classpath:helpers/utils.feature')
+    # auth
+    * def accessToken = karate.callSingle('classpath:auth/auth-redirect.feature').accessToken
+    * def requestHeaders = call read('classpath:auth/auth-headers.js')
+    * configure headers = requestHeaders 
+    * url baseURL
+  
+Scenario:  Healthcare worker can add deceased date and death notification
+    * def nhsNumber = '9000000033'
+    * path 'Patient', nhsNumber
+    * method get
+    * status 200
+    * def originalVersion = parseInt(response.meta.versionId)
+    * configure headers = call read('classpath:auth/auth-headers.js')     
+    * header Content-Type = "application/json-patch+json"
+    * header If-Match = karate.response.header('etag')
+    * def deceasedDate = '2020-01-01'
+    * def extensionVal = 
+    """
+      {
+        "extension": [
+          {
+            "url": "deathNotificationStatus",
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "code": "1",
+                  "display": "Informal - death notice received via an update from a local NHS Organisation such as GP or Trust",
+                  "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-DeathNotificationStatus",
+                  "version": "1.0.0"
+                }
+              ]
+            }
+          },
+          {
+            "url": "systemEffectiveDate",
+            "valueDateTime": "2020-02-27T16:14:58+00:00"
+          }
+        ],
+        "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-DeathNotificationStatus"
+      }
+    """
+    * path 'Patient', nhsNumber
+    * request 
+     """ 
+    {
+      "patches": [
+        { "op": "add", "path": "/deceasedDateTime", "value": "#(deceasedDate)" }, 
+         {"op": "add", "path": "/extension/-", "value": "#(extensionVal)" }
+    ]
+    } 
+    """
+    * method patch
+    * status 200
```

Comment:
Probably worthwhile adding some more assertions to this and the below scenario, i.e. checking the nhs number of the response, and that it has a deceasedDateTime ect.

Reply:
added deceasedDateTime  validation